scout/adapter/mongo/institute.py

```diff
@@ -174,7 +175,7 @@ def safe_genes_filter(self, institute_id):
             return safe_genes  # return an empty list
         for panel_name in institute_obj.get("gene_panels_matching", {}).keys():
             safe_genes += self.panel_to_genes(panel_name=panel_name, gene_format="hgnc_id")
```

Comment:
This is kind of already projected because panel_to_genes uses projection (the whole gene not the hgnc_id). But shouldn't matter in the end because this code is executed only when institute settings are updated

Reply:
Ahem, I think you might have a commit unpushed? I dont see the storing of that list anywhere? Also they probably have to update for all institutes when a panel gets updated.

Reply:
Yes, they need to update all institutes. The list gets saved in the institutes, see my other comment

Reply:
Right, but e.g. if I add a gene to the NMD panel and have it on "safe gene matching" it wouldn't update until I go to the institute page and save again?!

Reply:
Mm good point. We could trigger an update of institute's safe genes whenever a panel is updated

Reply:
Ok, I've included a safe gene update whenever you create/update/delete a gene panel. It's a quite ugly solution and I'm not a big fan of it myself, so if any ideas for improvement are welcome.